README.md

```diff
@@ -1206,6 +1206,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Luiz Barreto](https://github.com/luizpgb)
   - [Lukasz Przybylka](https://github.com/l-przybylka)
   - [Lundbyit](https://github.com/Lundbyit)
+  - [Laxman Gupta](https://github.com/laxman-gupta1006)
```

Comment:
Please add your name in line 1185, after Lavakush Biyani.
Let us know if you need any help with that. 🙂